requirements.txt

```diff
@@ -80,5 +80,5 @@ djangorestframework-guardian==0.3.0
 flake8==3.8.2 
 black==19.10b0
 quark-engine==20.8
-git+git://github.com/DissectMalware/XLMMacroDeobfuscator.git@c78b9e443e2667399470462fd863ebe4e2b8c978
+git+git://github.com/DissectMalware/XLMMacroDeobfuscator.git@89fbce0c87014a4b5a22c1aef09c8b3ea9bf16c0
```

Comment:
Can you replace this with the pip module ? https://pypi.org/project/XLMMacroDeobfuscator/

Reply:
Unfortunely the pip version is 0.1.4, and to deobfuscate the latest macro, 0.1.5 is required

Reply:
Yes, XLMMacroDeobfuscator is a very useful too but very experimental so we should try to get the latest version that we were able to properly test. That commit is ok imho and solves the problem that @0ssigeno stated